app/services/disclosure_log.rb

```diff
@@ -18,7 +18,11 @@ def import!
                    pluck(:reference)
     imported_refs = import.pluck(:reference)
     expired_refs = current_refs - imported_refs
-    PublishedRequest.where(reference: expired_refs).destroy_all
+
+    PublishedRequest.
+      where(case_management: case_management.name).
```

Comment:
Probably add a scope here so that we can do `PublishedRequest.from(case_management).where`

Reply:
Resolved by cf6b5f8